### PR TITLE
Add command for checking https sites with untrusted cert.

### PR DESCRIPTION
**Pull request by bostrt**

### CODE REVIEW

File: sopel/modules/isup.py [hunk 1]
```diff
@@ -25,7 +27,10 @@ def isup(bot, trigger):
         site += ".com"
 
     try:
-        response = web.get(site)
+        response = web.get(site, verify_ssl=secure)
+    except SSLError:
+        bot.say(site + ' looks down from here. Try using .isupinsecure')
```
Comment (dgw) [inline sopel/modules/isup.py]: Use `bot.config.core.help_prefix` instead of hard-coding `.` here.

File: sopel/modules/isup.py [hunk 2]
```diff
@@ -6,11 +6,13 @@
 from sopel import web
 from sopel.module import commands
 
+from requests.exceptions import SSLError
```
Comment (dgw) [inline sopel/modules/isup.py]: Swap this line with the blank above it, keeping the imports together and also keeping PEP8 happy with two blank lines before a method definition.

File: sopel/modules/isup.py [hunk 3]
```diff
@@ -5,12 +5,13 @@
 
 from sopel import web
 from sopel.module import commands
+from requests.exceptions import SSLError
 
-
```
Comment (dgw) [inline sopel/modules/isup.py]: Keep this blank line or flake8 will complain. (Soon, the test suite will enforce this for you; see #1278.)

**Comment by dgw** [comment]
@bostrt Once you've finished, please squash this down to the first commit with `rebase -i master` and prefix the commit message with "isup:" for the history. Thanks!

(The commit message might be long, maybe use "isup: add cmd to check untrusted HTTPS"?)

**Comment by bostrt** [comment]
@dgw sorry for the delay but I think all requests have been satisfied (code-style and squashing).

**Comment by dgw** [comment]
Indeed, thank you. There's a merge conflict now, but hold off on rebasing for the time being. #1316 will also require rebasing this module so you're best off waiting until that's done.

**Comment by dgw** [comment]
Say @bostrt, would it be easier for you if I rebased this myself?

**Comment by bostrt** [comment]
Sorry for the holdup, I can do the rebase in a bit.

**Comment by bostrt** [comment]
Hopefully all good now

**Comment by dgw** [comment]
Thanks for sticking with it, @bostrt!

**Pull request merged**